CMakeLists.txt

```diff
@@ -70,10 +70,31 @@ if(ENABLE_TESTS)
     add_subdirectory(tests)
 endif()
 
+#------------------------------------------------------------------------------
+# Configure examples
+#------------------------------------------------------------------------------
+
+add_subdirectory(examples)
+
 #------------------------------------------------------------------------------
 # Add Code Checks
 #------------------------------------------------------------------------------
 
 serac_add_code_checks(PREFIX   serac
                       INCLUDES /src/ /tests/
                       EXCLUDES cmake/blt uberenv_libs)
+
+#------------------------------------------------------------------------------
+# Export Targets
+#------------------------------------------------------------------------------
+add_library(serac INTERFACE)
```

Comment:
Did having two CMake targets named the same thing not cause an error? serac the executable and serac the library?